modules/accredited_representative_portal/app/sidekiq/accredited_representative_portal/send_poa_to_corp_db_job.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class SendPoaToCorpDbJob
```

Comment:
Maybe consider updating the name to something like `SendPoaRequestToCorpDbJob`?
That way we can help prevent confusion between actual POA submissions vs POA Requests.